tests/integration/dismiss-mem.tcl

```diff
@@ -47,12 +49,12 @@ start_server {tags {"dismiss external:skip"}} {
         r xadd bigstream * entry1 $bigstr entry2 $bigstr
 
         set digest [debug_digest]
-        r config set aof-use-rdb-preamble no
```

Comment:
i don't think we want to delete this line, but better move it to the server startup config.
I suppose the intent here was some coverage of the AOF rewrite mechanism.
maybe we should have also had here a `BGSAVE` and `DEBUG RELOAD NOSAVE` to cover RDB saving too?
@ShooterIT do you remember what was the intent here?

Reply:
> suppose the intent here was some coverage of the AOF rewrite mechanism.

yes, this test file just wants to 100% cover all changes.

the next case executes `BGSAVE` but doesn't execute `DEBUG RELOAD NOSAVE`